IceCubesApp/Resources/Localization/Plurals/fr.lproj/Localizable.stringsdict

```diff
@@ -18,21 +18,85 @@
 			<string>%lld nouveaux postes</string>
 		</dict>
 	</dict>
-	<key>notifications-others-count %lld</key>
-	<dict>
-		<key>NSStringLocalizedFormatKey</key>
-		<string>%#@noficationsOthersCount@</string>
-		<key>noficationsOthersCount</key>
-		<dict>
-			<key>NSStringFormatSpecTypeKey</key>
-			<string>NSStringPluralRuleType</string>
-			<key>NSStringFormatValueTypeKey</key>
-			<string>lld</string>
-			<key>one</key>
-			<string> et %lld autre </string>
-			<key>other</key>
-			<string> et %lld autres </string>
-		</dict>
-	</dict>
+    <key>notifications-others-count %lld</key>
+    <dict>
+        <key>NSStringLocalizedFormatKey</key>
+        <string>%#@noficationsOthersCount@</string>
+        <key>noficationsOthersCount</key>
+        <dict>
+            <key>NSStringFormatSpecTypeKey</key>
+            <string>NSStringPluralRuleType</string>
+            <key>NSStringFormatValueTypeKey</key>
+            <string>lld</string>
+            <key>one</key>
+            <string> et %lld autre </string>
+            <key>other</key>
+            <string> et %lld autres </string>
+        </dict>
+    </dict>
```

Comment:
You probably need to make an English version of that we can and then you can copy it to every other plurals dict